doc/cpp_recipe.rst

```diff
@@ -0,0 +1,230 @@
+Recipes
+===============
+
+An Arbor recipe is a description of a model without any information about how the
+model is to be executed. It  provides interface that the model building phase queries
+for information on cells, such as:
+
+  * the number of cells in the model;
+  * the type of a cell;
+  * a description of a cell;
+  * incoming network connections on a cell.
+
+The :cpp:class:`arb::recipe` class documentation is below.
+
+Why Recipes?
+--------------
+
+The Recipe design was motivated by the following aims:
+
+    * Building a simulation from a recipe description must be possible in a
+      distributed system efficiently with minimal communication.
+    * The model description (i.e. recipe) should be seperate from the
+      instantiation of the model to execute on the target system.
```

Comment:
Spelling: 'separate'. Point not clear: it reads as if the recipe should be separate in order to execute on the target system, but this doesn't really make sense.

Reply:
Line 22 seems to be deleted, but replacement text missing. Or was line 23 also meant to be removed?